Search/Adapter/ZendLuceneAdapter.php

```diff
@@ -162,8 +162,8 @@ public function index(Document $document, $indexName)
         $luceneDocument->addField(
             Lucene\Document\Field::unStored(self::AGGREGATED_INDEXED_CONTENT, implode(' ', $aggregateValues))
         );
-        $luceneDocument->addField(Lucene\Document\Field::unIndexed(self::URL_FIELDNAME, $document->getUrl()));
-        $luceneDocument->addField(Lucene\Document\Field::unIndexed(self::TITLE_FIELDNAME, $document->getTitle()));
+        $luceneDocument->addField(Lucene\Document\Field::text(self::URL_FIELDNAME, $document->getUrl()));
+        $luceneDocument->addField(Lucene\Document\Field::text(self::TITLE_FIELDNAME, $document->getTitle()));
```

Comment:
@chirimoya I think you have the most experience with running Zend Lucene in a production environment from us. Do you think that has a big performance impact?